documentation/chatmeleon-swagger.yaml

```diff
@@ -161,37 +162,40 @@ paths:
                     type: string
                     pattern: "^[0-9a-f]{24}$"
                     description: The ID of the earliest message of the current messages page.
+        "401":
+          description: Unauthorized - JWT token is missing or invalid.
+        "403":
+          description: Forbidden - User not allowed to retrieve the messages of the conversation.
+        "404":
+          description: Not Found - Conversation not found.
   /messages:
     post:
       description: Create a new message.
       tags:
         - messages
       summary: Create a message
+      security:
+        - bearerAuth: []
       responses:
         "200":
           description: Message created successfully.
-  /users/{userId}:
-    get:
-      description: Retrieve information about a user identified by the user's ID.
-      tags:
-        - users
-      summary: Get a user
-      parameters:
-        - name: userId
-          in: path
-          description: The unique user ID.
-          required: true
-          schema:
-            type: string
-          example: "3564f38ce881480099dd7cb1"
-      responses:
-        "200":
-          description: User's information.
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/User"
+                $ref: "#/components/schemas/Message"
```

Comment:
Why this route response with a Message after creating a Message per request?

Reply:
I was asking the same thing but tbh this is something I missed out on when I PR reviewed this back when @XkhangNguyen and @KhoaNgo18 finished this, so can you two help me answer this pls lol